src/app/Scenes/MyProfile/Components/UserAccountHeader/ErrorMessageComponent.tsx

```diff
@@ -0,0 +1,7 @@
+import { SimpleMessage } from "@artsy/palette-mobile"
+
+export const ErrorMessageComponent = (
```

Comment:
The name is pretty universal, but the error message is related to the profile. I'd suggest reflecting this in the name of the component.

Also, `...Component` is usually a pattern we do not use because the type and the name starting with a capital letter indicate that it's a component.

```suggestion
export const ProfileErrorMessage = (
```